### ISSUE TITLE
Add string pattern found count to find/replace tools.

### ISSUE THREAD

**Issue by aspenforest** [godotengine/godot#30286]
When I refactor my code using the find and replace tools, it would be extremely helpful to know how many instances of the search string the tool has found.   

Sometimes I need to create const string references in a huge source file and there is no point in making them if the string is used only once, so I create the constant strings only if there are multiple instances of the string used.  

I _could_ use the "up" or "down" arrow to check if the editor jumps to another instance if there exists one, but that's not an elegant solution...

**Comment by Paulb23** [comment]
Duplicate of #14513?

**Comment by bojidar-bg** [comment]
Indeed, closing as a duplicate of the above-mentioned #14513.
